webapp/src/components/hiddenCardCount/hiddenCardCount.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react'
+import {useIntl} from 'react-intl'
+
+import Button from '../../widgets/buttons/button'
+
+import {Card} from '../../blocks/card'
+import './hiddenCardCount.scss'
+
+type Props = {
+    hiddenCards: Card[]
```

Comment:
Didn't notice before, but you are only using the `.length` of array. Probably better to just get and pass an `Integer` rather than a `Card[]`.